src/ResourceManager/ServiceBus/Commands.ServiceBus/Utilities/ServiceBusClient.cs

```diff
@@ -541,6 +540,55 @@ public bool DeleteRule(string resourceGroupName, string namespaceName, string to
 
         #endregion Rules
 
+        #region DRConfiguration
+        public ServiceBusDRConfigurationAttributes GetServiceBusDRConfiguration(string resourceGroupName, string namespaceName, string alias)
+        {
+            var response = Client.DisasterRecoveryConfigs.Get(resourceGroupName, namespaceName, alias);
+            Thread.Sleep(TimeSpan.FromSeconds(5));
```

Comment:
@v-Ajnava why do these methods need `Thread.Sleep`?

Reply:
Removed, was used to wait till cache is refreshed.